dev/halyard_install_component.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
This is meant to be used as a part of our build process, so I think it belongs in /dev.